simulation/traffic_simulator/src/math/hermite_curve.cpp

```diff
@@ -193,7 +201,7 @@ boost::optional<double> HermiteCurve::getCollisionPointIn2D(
       double poly_x = (1 - tx) * point1.x + tx * point0.x;
       double poly_y = (1 - ty) * point1.y + ty * point0.y;
       double error = std::hypot(poly_x - x, poly_y - y);
-      if (error < 3 && 0 < tx && tx < 1 && 0 < ty && ty < 1 && 0 < solution && solution < 1) {
+      if (error < 0.1) {
```

Comment:
Please leave a comment for the reason for this magic number.

Reply:
fixed in 7da4bc96c3a07a367967a2f1f797974c3855c5d7

Reply:
Fixed in 20775b94747b8d1cd86c11ba7002b2713764f6ce